_posts/2019-02-18-event-store-5.0.0-release.md

```diff
@@ -0,0 +1,193 @@
+* [#1801](https://github.com/EventStore/EventStore/pull/1801) - **(Test Client)** Use `testclient.conf` for the EventStore test client instead of `eventstore.conf`.
+
+### Performance
+* [#1756](https://github.com/EventStore/EventStore/pull/1756) - **(Server)** Faster node initialization  
+A new config option has been added: `InitializationThreads` to specify number of threads used to load chunk and index files. This can have a significant impact on startup times. (thanks to [@megakid](https://github.com/megakid)!)
+* [#1639](https://github.com/EventStore/EventStore/pull/1639) - **(Server/Windows)** Add option to reduce pressure on the system file cache  
+A new config option has been added: `ReduceFileCachePressure` that disables the `FileOptions.RandomAccess` cache flag when opening chunk files. This is particularly worth trying if the Windows File Cache is holding up a huge amount of memory. (thanks to [@lscpike](https://github.com/lscpike)!)
+* [#1637,1638](https://github.com/EventStore/EventStore/pull/1638) - **(Scavenging/Performance)** Faster scavenges (up to ~10x speedup!) (thanks to [@lscpike](https://github.com/lscpike) and [@megakid](https://github.com/megakid)!)
+* [#1640](https://github.com/EventStore/EventStore/pull/1640) - **(Server)** Fix latency issue in index committer service queue & off by one error when shutting down EventStore services
+* [#1688](https://github.com/EventStore/EventStore/pull/1688) - **(Server)** Batch log read expired messages when a high number of reads are expiring
+* [#1829](https://github.com/EventStore/EventStore/pull/1829) - **(Server)** Do not publish `StorageMessage.EventCommitted` messages when rebuilding the index. This speeds up full index rebuilds by a factor of approximately 1.8x and also speeds up node startup time by a few seconds when there are many index entries to rebuild.
+* **[Packaging/Performance]** Add `--optimize=inline` mono runtime optimization to native packages
+
+### Miscellaneous
+* [#210](https://github.com/EventStore/EventStore.UI/pull/210) - **(UI)** Remove bower components from repository
+* [#1823](https://github.com/EventStore/EventStore/pull/1823) - **(UI)** Install bower for UI builds
+* [#1796](https://github.com/EventStore/EventStore/pull/1796) - **(API Docs)** Polish ExpectedVersion docs
+* [#1800](https://github.com/EventStore/EventStore/pull/1800) - **(Server)** Add default file locations for macOS (for system-wide installations)
+* [#1643](https://github.com/EventStore/EventStore/pull/1643) - **(Server)** Clean up persistent subscription logging (thanks to [@lscpike](https://github.com/lscpike)!)
+* [#1629](https://github.com/EventStore/EventStore/pull/1629) - **(Server)** Clean up scavenge log (thanks to [@lscpike](https://github.com/lscpike)!)
+
+## Where can I get the packages?
```

Comment:
@shaan1337 I kind of feel like this should be nearer the top.

Reply:
In fact all of this section down until the end, likely the most important parts that people will want to read and all are right at the bottom of a very long document.